addons/a11y/userscript.js

```diff
@@ -198,21 +99,9 @@ export default async function (o) {
   async function renderLoop() {
     while (true) {
       cleanUp();
-      for (let e of Object.entries(a11yObjects)) {
-        updateAria(vm.runtime.targets.filter((a) => a.id === e[0])[0]);
+      for (let e in a11yObjects) {
```

Comment:
a11y updates in another PR please

Reply:
They aren't updates. It just make it use the new blocks api. Same for pause, tho I did add the breakpoint block. (Same as sa-pause)

Reply:
@easrng but wasn't the code that handled a11y blocks commented/impossible to run? why are you updating code that will not run?

Reply:
> we are phasing sa- out

Why?

Reply:
> wasn't the code that handled a11y blocks commented/impossible to run?

The code to add the blocks to the editor was disabled but it always processed the block runs. If you want to hide the a11y blocks from the flyout again that will be easy to do.

Reply:
> > we are phasing sa- out
> 
> Why?

Idk, it was discussed in the discord.